examples/cpem-add-on/main.tf

```
@@ -3,17 +3,18 @@ module "tfk8s" {
   # source  = "equinix-labs/terraform-equinix-kubernetes-cluster?"
   # version = "0.0.1" # Use the latest version, according to https://github.com/equinix-labs/terraform-equinix-kubernetes-cluster/releases
 
-  auth_token         = var.auth_token
-  kube_vip_version   = var.kube_vip_version
-  kubernetes_version = var.kubernetes_version
-  project_id         = var.project_id
-  cp_ha              = var.cp_ha
-  worker_count       = var.worker_count
-  metro              = var.metro
+  metal_auth_token        = var.metal_auth_token
+  kube_vip_version        = var.kube_vip_version
+  kubernetes_version      = var.kubernetes_version
+  metal_project_id        = var.metal_project_id
+  cp_ha                   = var.cp_ha
+  worker_count            = var.worker_count
+  metro                   = var.metro
+  cloud_provider_external = ""
```

this should come from the variable definition
```suggestion
  cloud_provider_external = var. cloud_provider_external
```